wistiavideo/static/html/wistiavideo.html

```diff
@@ -0,0 +1,9 @@
+<div class="wistiavideo_block">
+  <script src="//fast.wistia.com/embed/medias/{self.media_id}.jsonp" async></script>
```

Comment:
@z4y4ts I took a quick look at https://wistia.com/doc/construct-an-embed-code#embed_code_types.  I didn't take the time yet to fully understand it.  Can you explain which type of embed code the XBlock supports, and if it does not support some types?  Are there fallbacks for support by different browsers?